src/main/java/com/example/model/Auth.java

```diff
@@ -10,7 +11,40 @@
 @Data
 @AllArgsConstructor
 public class Auth {
-  private int userId;
-  private String authToken;
-  private Date createTime;
+  public int userId;
```

Comment:
The "@Data" annotation automagically creates getters and setters for the data members. The "@AllArgsConstructor" also automatically creates a constructor with all the args so you don't need that either. The other constructors you can add but I would take the rest out.

Reply:
Good catch. Fixed